HikingApp.MacCatalyst/ViewControllers/MainViewController.cs

```diff
@@ -0,0 +1,31 @@
+using UIKit;
+
+namespace HikingApp.MacCatalyst.ViewControllers
+{
+    public class MainViewController : UITabBarController
+    {
+        public override void ViewDidLoad()
+        {
+            base.ViewDidLoad();
+
+            var exploreViewController = new ExploreViewController();
+            var myTrailsViewController = new MyTrailsViewController();
+
+            var exploreNavigationController = new UINavigationController(exploreViewController)
+            {
+                TabBarItem = new UITabBarItem("Explore", UIImage.FromBundle("explore_icon.png"), 0)
```

Comment:
Do we have the `explore_icon.png` somewhere?

Reply:
No, let's remove them.